joplin/api/schema.py

```diff
@@ -281,7 +393,7 @@ class Language(graphene.Enum):
     BURMESE = 'my'
 
 
-class ServicePageStepLocationBlock(graphene.ObjectType):
+class StreamFieldLocationPage(graphene.ObjectType):
```

Comment:
this is a name change unrelated the event content type, no? I feel like the old name made sense. Does this need to be a part of this PR?

Reply:
I was going to try to reuse it for city_location on event page but forgot it was actually building an array, forgot to undo the rename when I decided not to go that route. I'm fixing it rn.